sdk/PowerBI.Core/Cryptography/AsymmetricKeyEncryptionHelper.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml.Serialization;
+
+namespace Microsoft.PowerBI.Core.Cryptography
+{
+    /// <summary>
+    /// Helper class for encrypting and decrypting with asymmetric keys
+    /// </summary>
+    public static class AsymmetricKeyEncryptionHelper
+    {
+        private const int SegmentLength = 85;
```

Comment:
These are the same fixed sizes that are used in DM.  Do we have a use case where we would need to support variable sizes?